bin/cdk-infra.ts

```diff
@@ -10,16 +10,25 @@ import FhirWorksStack from '../lib/cdk-infra-stack';
 const app = new cdk.App();
 
 const allowedLogLevels = ['error', 'info', 'debug', 'warn'];
+const allowedFHIRVersions = ['4.0.1', '3.0.1']
 
 const region: string = app.node.tryGetContext('region') || 'us-west-2';
 const stage: string = app.node.tryGetContext('stage') || 'dev';
 const enableMultiTenancy: boolean = app.node.tryGetContext('enableMultiTenancy') || false;
 const enableSubscriptions: boolean = app.node.tryGetContext('enableSubscriptions') || false;
 const oauthRedirect: string = app.node.tryGetContext('oauthRedirect') || 'http://localhost';
 const useHapiValidator: boolean = app.node.tryGetContext('useHapiValidator') || false;
-let logLevel: string = app.node.tryGetContext('logLevel') || 'error';
 const enableESHardDelete: boolean = app.node.tryGetContext('enableESHardDelete') || false;
 const enableBackup: boolean = app.node.tryGetContext('enableBackup') || false;
+let logLevel: string = app.node.tryGetContext('logLevel') || 'error';
+let fhirVersion: string = app.node.tryGetContext('fhirVersion') || '4.0.1';
+
+if (useHapiValidator) {
+    if (!allowedFHIRVersions.includes(fhirVersion)) {
+        console.log(`invalid FHIR Version specified: ${fhirVersion}`);
+        fhirVersion = '4.0.1';
```

Comment:
Should we fail the deployment directly if the fhir version specified is not allowed?

Reply:
This makes more sense, changing this to throw an error instead.